pkg/kustomize/kustomize.go

```diff
@@ -618,11 +618,19 @@ func Inflate(
 		resources[index] = obj
 	}
 
-	for index, resource := range resources {
-		resources[index] = v1.EnsureOwnerReference(quay, resource)
+	var filteredResources []client.Object
+	for _, resource := range resources {
+		if resource == nil {
+			continue
+		}
+		filteredResources = append(filteredResources, resource)
+	}
```

Comment:
couldn't we do this in one pass?
```suggestion
	var filteredResources []client.Object
	for _, resource := range resources {
		if resource == nil {
			continue
		}
		res, err := v1.EnsureOwnerReference(quay, resource)
		if err != nil {
		    log.Error().Err(err).Msg(...)
		    continue
		   }
		   
		filteredResources = append(filteredResources, res)
	}
	
	return filteredResources, err
```